pkg/proto/datadog/trace/remote_rates.proto

```diff
@@ -0,0 +1,25 @@
+syntax = "proto3";
```

Comment:
This shows up as a new file, while others were moved. Is this git just failing to detect a rename?